examples/gnrc_border_router/Makefile.wifi.conf

```diff
@@ -0,0 +1,7 @@
+# Set the SSID and password of your WiFi network here
+WIFI_SSID ?= "Your_WiFi_name"
+WIFI_PASS ?= "Your_secure_password"
+
+CFLAGS += -DESP_WIFI_SSID=\"$(WIFI_SSID)\"
+CFLAGS += -DESP_WIFI_PASS=\"$(WIFI_PASS)\"
+CFLAGS += -DESP_WIFI_PASS=\"Your_secure_password\"
```

Comment:
This should be removed.

Reply:
Yeah of course, I had assumed that @benpicco knows how it was meant. I haven't figured out how multiline suggestions work in GitHub :dark_sunglasses: